<?xml version="1.0"?>
<launch>
    <node pkg="vcpd" name="tsdf_node" type="tsdf_fusion.py" output="screen" />
        <rosparam command="load" file="$(find vcpd)/config/tsdf_params.yaml">
        <rosparam command="load" file="$(find vcpd)/config/rs_params.yaml">
</launch>
